src/main/scala/mesosphere/marathon/core/task/termination/impl/KillStreamWatcher.scala

```diff
@@ -40,17 +40,13 @@ object KillStreamWatcher extends StrictLogging {
       merge(killedViaUnknownInstanceTerminated, eagerComplete = true)
   }
 
-  @SuppressWarnings(Array("AsInstanceOf"))
-  def ignoreFlow[T]: Flow[T, Nothing, NotUsed] =
-    Flow[T].filter(_ => false).asInstanceOf[Flow[T, Nothing, NotUsed]]
-
   private val singleDone = List(Done)
   private[impl] def killedInstanceFlow(instanceIds: Iterable[Instance.Id]): Flow[Instance.Id, Done, NotUsed] = {
 
     val instanceIdsSet = instanceIds.toSet
     if (instanceIdsSet.isEmpty) {
       logger.info("Asked to watch no instances. Completing immediately.")
-      ignoreFlow[Instance.Id].prepend(Source.single(Done)).take(1)
+      EnrichedFlow.ignore[Instance.Id].prepend(Source.single(Done)).take(1)
```

Comment:
Is there a reason why we can't write `Source.single(Done)` or `Source.empty[Instance.Id]`?